test/data/notificationSigned.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "c9377529-d7df-4aa1-ae37-ad5148612003",
+  "source_transfers":[{
+    "id": "http://usd-ledger.example/USD/transfers/6851929f-5a91-4d02-b9f4-4ae6b7f1768c",
+    "ledger":"http://usd-ledger.example/USD",
+    "debits":[{
+      "amount":"1.07",
+      "account":"http://usd-ledger.example/accounts/alice"
+    }],
+    "credits":[{
+      "amount":"1.07",
+      "account":"http://usd-ledger.example/accounts/mark"
+    }],
+    "execution_condition": {
+      "message_hash": "9EXcGgJa0zTIUxEQXaMKpvoGsNmYSxv8J6C12LBNA5hRMfmbwDCzAU4O14UKnqcIaQqMGULHLI+T6Nk3+qYd3Q==",
+      "signer": "http://eur-ledger.example/EUR",
+      "type": "ed25519-sha512",
+      "public_key": "Lvf3YtnHLMER+VHT0aaeEJF+7WQcvp4iKZAdvMVto7c="
+    },
+   "expires_at": "2015-06-16T00:00:02.000Z",
+    "state": "prepared"
+  }],
+  "destination_transfers":[{
+    "id": "http://eur-ledger.example/EUR/transfers/c92f2a2c-b21d-4e6c-96e7-4f6d6df4bee9",
+    "ledger":"http://eur-ledger.example/EUR",
+    "debits":[{
+      "amount":"1.00",
+      "account":"http://eur-ledger.example/accounts/mark"
+    }],
+    "credits":[{
+      "amount":"1.00",
+      "account":"http://eur-ledger.example/accounts/bob"
+    }],
+    "state": "proposed"
+  }]
```

Comment:
This looks like a payment that is not signed, maybe it should be called something other than `notificationSigned`